config/applications.json

```diff
@@ -727,6 +727,14 @@
 		"link": "https://git-scm.com/",
 		"winget": "Git.Git"
 	},
+	"gitbutler": {
+		"category": "Development",
+		"choco": "na",
+		"content": "Git GUI",
```

Comment:
The "content" is the name that the application is displayed as, may I ask why it is named "Git GUI" and not  "GitButler"?

Reply:
Yeah it should be, Thanks for spotting it.